pro_1.py

```diff
@@ -0,0 +1,100 @@
+import json
+import Patterns
+import config_st
+import database_interaction
+import User
+import re 
+from re import findall
+database_file=open("database.txt","r+")
+dict_n=json.load(database_file)
```

Comment:
this should be part of the database_interaction module

Reply:
i explain it up